src/core_modules/capture-core/components/Widget/widget.types.js

```diff
@@ -1,10 +1,17 @@
 // @flow
+import type { WidgetCollapsibleProps } from './widgetCollapsible.types';
+import type { WidgetNonCollapsibleProps } from './widgetNonCollapsible.types';
 
-export type Props = {|
-    header: React$Node,
-    children: React$Node,
-    open: boolean,
-    onOpen: () => void,
-    onClose: () => void,
-    ...CssClasses,
+type CollapsibleProps = {|
+    ...WidgetCollapsibleProps,
+    color?: ?string,
```

Comment:
`color` is part of the `WidgetCollapsibleProps` so we don't need to readd it here. Same below.